packetery/template/order/order-detail.latte

```diff
@@ -0,0 +1,30 @@
+{varType \PacketeryModule\Order\Entity $order}
+
+<section class="packetery-order-detail">
+
+	<h2 class="woocommerce-order-details__title">{__( 'Packeta', 'packetery' )} - {__( 'selectedPickupPoint', 'packetery' )}</h2>
+
+	<table class="woocommerce-table woocommerce-table--order-details shop_table order_details">
+		<tbody>
+		<tr>
+			<th scope="row">{__( 'pickupPointName', 'packetery' )}</th>
+			<td>
+				{$order->getPointName()}
+
+				{if $order->getCarrierId() === 'packeta'}
```

Comment:
Konstanta pro 'packeta'

Reply:
více by se mi líbila metoda isInternalCarrier(). Taková metoda má potenciál vícenásobného použití a je to více zjevné na co se tady ptáš.

Reply:
Co !$order->isExternalCarrier() ?

Reply:
V zadání je zmíněné, že se odkaz má zobrazovat jen když je objednávka nasměrována na interní výdejní místo. Ta navrhovaná podmínka je matoucí v případě, že objednávka bude nasměrovaná na PacketaHD

Já vnímám externího dopravce jako dopravce který nesouvisí se zásilkovnou.

Reply:
Důležité je, že naše systémy jako externího doprace berou všechny dopravce, kteří mají numerické id. Nicméně metodu jsem pro přehlednost přidal.